docs/docs-content/vm-management/create-manage-vm/using-datavolumes.md

```diff
@@ -0,0 +1,184 @@
+---
+sidebar_label: "Using DataVolumes for disk templates"
+title: "Using DataVolumes for disk templates"
+description: "Using DataVolumes to quickly create VMs from template disks in Palette Virtual Machine Orchestrator"
+icon: " "
+hide_table_of_contents: false
+sidebar_position: 20
+tags: ["vmo"]
+---
+
+
+
+When VMs are deployed from template, they almost always include a disk template containing a preinstalled copy of the desired OS. The Kubevirt CDI is used to import these templates through a concept called [Data Volumes](https://github.com/kubevirt/containerized-data-importer/blob/main/doc/datavolumes.md) and it supports multiple different approaches to do so. Some approaches are vastly quicker than others though. Hence, designing this correctly is important. This page details how to do that.
+
+
+## Overview
+
+![Image cloning with CDI](/cdi-disk-clone.png)
+
+The diagram above is a representation of best approach:
```

Comment:
I don't own this diagram, I got it from the web as a picture. Turning the 4 steps into diagram becomes messy because it would be 4 boxes that all reference other things. It would get just as wordy as the text below.

Reply:
I adjusted the wording a bit